scripts/install_asi_libs.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -eu
+
+export CAM_LIB_VERSION=1.15.0617
+export EFW_LIB_VERSION=1.5.0615
+
+export BUILD_DIR=$HOME/ASIBuild
```

Comment:
Building something off the home directory kind of pollutes it. Suggest using `/tmp/`:
```suggestion
export BUILD_DIR=/tmp/ASIBuild
```

Reply:
@lspitler  Good point, although I'm not sure that we can always rely on `/tmp` existing in the docker images (I had that problem before). We probably want to get rid of the build dir anyway to save space on the pis, so instead we can just delete it?